src/screens/IndexScreen.js

```diff
@@ -50,6 +51,17 @@ const getListItems = (query, data) => {
               (eventRecord.addresses[0].addition || eventRecord.addresses[0].city)
           ),
           title: eventRecord.title,
+          picture: {
+            url:
+              !!eventRecord.mediaContents &&
+              !!eventRecord.mediaContents.length &&
+              _filter(
+                eventRecord.mediaContents,
+                (mediaContent) =>
+                  mediaContent.contentType === 'image' || mediaContent.contentType === 'thumbnail'
+              )[0].sourceUrl &&
+              !!eventRecord.mediaContents[0].sourceUrl.url,
```

Comment:
you can use `mainImageOfMediaContents(eventRecord.mediaContents)` i think, like for tours for example. than this gets shorter here. the helper is already existing.

Reply:
last line returns true or false with using !!...
with using the helper, this will be fixed as well and images should become visible.